packages/rocketchat-ui-admin/client/users/adminUsers.js

```diff
@@ -131,18 +131,20 @@ Template.adminUsers.onRendered(function() {
 	});
 });
 
+const DEBOUNCE_TIME_FOR_SEARCH_USERS_IN_MS = 300;
+
 Template.adminUsers.events({
 	'keydown #users-filter'(e) {
 		if (e.which === 13) {
 			e.stopPropagation();
 			e.preventDefault();
 		}
 	},
-	'keyup #users-filter'(e, t) {
+	'keyup #users-filter': _.debounce((e, t) => {
```

Comment:
should only the the `t.filter.set()` be debounced? I wonder because the event is not having its propagation stopped nor default prevented...

Reply:
@sampaiodiego if we debounce this this function too, for some reason the event is not prevented, and the form is submitted.